src/main/java/Array17.java

```diff
@@ -0,0 +1,44 @@
+public class Array17 {
+
+    public int[][] setSpiralArray(int n) {
+        int[][] result = new int[n][n];
+        int startRow = 0;
+        int startColumn = 0;
+        int row = n;
+        int column = n;
+        int value = 1;
+
+        while ((startRow - row) + (startColumn - column) < 0) {
+
+            for (int i = startColumn; i < column; i++) {
+                result[startRow][i] = value;
+                value++;
+            }
+            startRow++;
+
+            for (int i = startRow; i < row; i++) {
```

Comment:
Dưới có if mà đây không có?

Reply:
2 vòng for đâu dùng đk ở khi bắt đầu vào vòng while.
sau 2 vòng đầu thì 2 biến so sánh ở vòng while nó thay đổi nên 2 vòng for sau e đặt if

Reply:
Done